virtme_ng/run.py

```diff
@@ -788,7 +788,7 @@ def _make_remote(self, args, make_command):
             tmp.write(
                 REMOTE_BUILD_SCRIPT.format(
                     args.build_host_exec_prefix or "",
-                    make_command + " -j$(nproc --all)",
+                    " ".join(make_command + ["-j$(nproc --all)"]),
```

Comment:
It feels wrong to use `" ".join()`, which will remove quoted words. Instead, we could use `shlex.join()`, but it will add a single quote around `-j$(nproc --all)` on purpose.

What about:

    shlex.join(make_command) + " -j$(nproc --all)"

Reply:
(Here as well, do you prefer not to change? I'm confused by your :+1: :) )

```suggestion
                    shlex.join(make_command) + " -j$(nproc --all)",
```

Reply:
I think I switched to that in the new push. Did I miss something?

Reply:
Strange, it looks like you did (forced) push, but not modifying that.

Reply:
I'm dumb, I forgot to squash the changes into the commit... should be good now. :sweat_smile: